managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+
+#### Connecting Fivetran to your Managed Database for PostgreSQL
+
+1. From the Fivetran user interface, click **Add destination** in the [destinations list](https://fivetran.com/dashboard/destinations).
+
+2. Enter a name for the destination (e.g. `my-new-database`), then click **Add**.
+
+3. Select **PostgreSQL**, then click **Select**.
+
+4. Fill in the required information to connect to your existing database:
+
+    - **Host**
+    - **Port**
+    - **Database Name**
+    - **Username**
+    - **Password**
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+5. Select **Connect directly** as **Connection method**.
+
+6. Select **Detect Changes via XMIN** as **Update Method** (other methods are not supported).
+
+    <Message type="note">
+    If you have set up IP address access restrictions from your IAM policies, copy the addresses from Fivetran IPs and edit your IAM policies to whitelist them.
+    </Message>
+
+7. Click **Save & Continue**.
+
+8. Select the certificate in the list corresponding to `CN=sdb.{region}.scw.cloud`, check its validity, and confirm your selection by clicking **Confirm**. 
+
+    <Message type="tip">
+    You can ensure the certificate is fully valid only if all of the following checks are true:
+      - Origin is Let's Encrypt
+      - The CN field is `sdb.{region}.scw.cloud`
+      - The complete certificate value is the same as the one hosted on `sdb.{region}.scw.cloud`
+      - The root certificate is the same as [Let's Encrypt ISRG X1 Root certificate](https://letsencrypt.org/certs/isrgrootx1.pem)
+    </Message>
+
+#### Connecting Fivetran to your existing database
+
+1. From the Fivetran user interface, click **Add connector** in the [connectors list](https://fivetran.com/dashboard/connectors).
+
+2. Select your destination (for instance `my-new-database`), then click **Select**.
+
+3. Select **PostgreSQL**, then click **Set up**.
+
+4. Fill in the required information to connect to your existing database:
+
+    - **Host**
+    - **Port**
+    - **Database Name**
+    - **Username**
+    - **Password**
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
```

Comment:
Managed PostgreSQL is not integrated with IAM yet so we need to replace this with "the username & password that you defined upon Database Instance Creation"